modules/opensimplex/noise.h

```diff
@@ -0,0 +1,62 @@
+#ifndef OPENSIMPLEX_NOISE_H
+#define OPENSIMPLEX_NOISE_H
+
+#include "image.h"
+#include "open-simplex-noise.h"
+#include "scene/resources/texture.h"
+#include <reference.h>
```

Comment:
Godot headers are included with `""`, `<>` is for system/thirdparty headers. And `core/` includes should be absolute. So this should be:
```cpp
#include "core/image.h"
#include "core/reference.h"
#include "scene/resources/texture.h"

#include <open-simplex-noise.h>
```